<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño del Mapa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditWidth">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>35</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHeight">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>65</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancho :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Alto :</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollAreaDisplay">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Op de Gen aleatoria </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditOffset">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Desplazamiento :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Octavas :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFrequencyScale">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxOctaves">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Freq. Aleatoria :</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxPersistence">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Persistencia :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de ruido :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxNoiseType">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>79</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Perlin Noise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Simplex Noise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Voronoi Noise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ridged Multifractal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Billowy Noise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Domain Warping</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño del Pincel</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderBrushSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Pincel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBrushMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Levantar / Suavizar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Suavizar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aplanar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ruido</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rellenar</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fuerza del Pincel</string>
     </property>
    </widget>
    <widget class="QSlider" name="sliderBrushIntensity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderFillColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelFillColorPreview">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Color del Pincel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Op de Pincel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
